sdk/src/builder.rs

```diff
@@ -2939,6 +2942,53 @@ mod tests {
         assert_eq!(parent.assertions().len(), 1);
     }
 
+    #[c2pa_test_async]
+    async fn test_redaction_async() {
```

Comment:
Did you mean to include this test case in this PR?

Reply:
Yes. There were no tests for the async codepath.